Dshop/apps/users/templates/users/registration.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block stripe %}
+    <script src="https://js.stripe.com/v3/"></script>
+    <script src="{% static 'main.js' %}"></script>
+    <script defer src="https://use.fontawesome.com/releases/v6.4.0/js/all.js"></script>
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="d-flex justify-content-center">
+            <div class="mx-auto col-10 col-md-8 col-lg-6 vh-100 mt-lg-3">
+                <form method="POST" action="">
```

Comment:
Important remark. If you go manually over the form fields like here (not in loop or form.as_p or similar), every time a field is added to the form you also need to go html template. This is adding more work, so loop might be preferred.
Another thing is that you are missing things in every form:
- non field errors https://docs.djangoproject.com/en/4.2/ref/forms/api/#django.forms.Form.non_field_errors
- any hidden fields

Please reconsider either using
- automatic forms like form.as_p 
- or loop over form fields like here: https://docs.djangoproject.com/en/dev/topics/forms/#looping-over-the-form-s-fields